dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers.test.ts

```diff
@@ -51,10 +63,38 @@ describe('get-utilisation-report-reconciliation-details-by-id.controller helpers
       });
     });
 
-    it('maps the fee record entity to the reconciliation details fee record item when the currencies do not match', () => {
+    it('converts the fees paid to ukef for the period to the reported fees when the payment currencies match', () => {
```

Comment:
I don't think any conversion happens in this case, we map the values like for like no?